src/hearings/services/validators.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+import { AbstractControl, ValidatorFn } from '@angular/forms';
+import * as moment from 'moment';
+import { HearingDateEnum } from '../models/hearings.enum';
+
+@Injectable({ providedIn: 'root' })
+export class ValidatorsService {
```

Comment:
When talking about service it will be a layer communicate with back end, like http service.
Could this put to a new folder src/hearings/utils, and rename to validators.utils.ts?